src/cmd/structs.rs

```diff
@@ -408,6 +408,33 @@ pub struct SVCSpec {
     pub svc_type: String,
 }
 
+#[derive(Default, Debug, Clone, PartialEq, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct PDB {
```

Comment:
Can you please add a deserialize test please ? We've seen some issues lately when bumping kubectl version, it would allow us preventing that from happening in the future.